library/camus2/releaseurl.ts

```diff
@@ -1 +1 @@
-export const release_url: string = "https://github.com/helix-collective/camus2/releases/download/0.17/camus2.x86_64-linux.gz -O /opt/bin/camus2.gz";
+export const release_url: string = "https://github.com/helix-collective/camus2/releases/download/0.16/camus2.x86_64-linux.gz -O /opt/bin/camus2.gz";
```

Comment:
@paul-thompson-helix  @timbod7  @barryskal  @bacek   do you guys remember why would plp downgrade camus version?

Reply:
This is a shared submodule, not plp code? So a revert here ultimately affects all users of this module when they update.